lib/commands/addOn.js

```diff
@@ -102,79 +110,86 @@ addOn.chooseRegionPrompt = function (addon, callback) {
-    });
+    print(Util.format('Select %s\n', addon.regions[result.region - 1].data));
+    callback(null, addon.regions[result.region - 1]);
+  });
 };
 
-//-----------------------------------------------------------------------------
+// ----------------------------------------------------------------------------
 // Prints a list of Add-Ons
-//-----------------------------------------------------------------------------
-addOn.printList = function(projectName, addons) {
-  modulus.io.print('Add-Ons provisioned for ' + projectName.verbose);
+// ----------------------------------------------------------------------------
+addOn.printList = function (projectName, addons) {
+  var addonConfig,configItem, i;
+  print('Add-Ons provisioned for ' + projectName.verbose);
 
-  if(addons instanceof Array === false || addons.length === 0) {
-    modulus.io.print('No Add-Ons provisioned.');
+  if (addons instanceof Array === false || addons.length === 0) {
+    print('No Add-Ons provisioned.');
   } else {
-    for(var a = 0; a < addons.length; a++) {
-      modulus.io.print('--------------------------------------'.grey);
+    for (i = 0; i < addons.length; ++i) {
+      print('--------------------------------------'.grey);
 
       // Capitalized name.
-      modulus.io.print(addons[a].addon_name.yellow);
+      print(addons[i].addon_name.yellow);
 
       // Id
-      modulus.io.print(('Id: ' + addons[a].addon_id).grey);
+      print(('Id: ' + addons[i].addon_id).grey);
 
       // Plan
-      modulus.io.print(('Plan: ' + addons[a].plan).grey);
+      print(('Plan: ' + addons[i].plan).grey);
 
       // Config
-      for(var k in addons[a].config) {
-        modulus.io.print((k + ' = ' + addons[a].config[k]).grey);
+      addonConfig = addons[i].config;
+      for (configItem in addonConfig) {
+        if ({}.hasOwnProperty.call(addonConfig, configItem)) {
+          print((configItem + ' = ' + addonConfig[configItem]).grey);
+        }
       }
     }
   }
 };
 
-//-----------------------------------------------------------------------------
+// ----------------------------------------------------------------------------
 // Gets the Add-Ons provided to a project.
-//-----------------------------------------------------------------------------
-addOn.getForProject = function(projectName, callback) {
+// ----------------------------------------------------------------------------
+addOn.getForProject = function (projectName, callback) {
   var project = null;
 
-  async.waterfall([
-    function(cb) {
+  Async.waterfall([
+    function (cb) {
       addOn.getProject(projectName, cb);
     },
 
-    function(pro, cb) {
+    function (pro, cb) {
       project = pro;
-      addOnController.getForProject(project.id, cb);
+      AddOnController.getForProject(project.id, cb);
     },
 
-    function(addons, cb) {
-      //Newline
-      console.log();
+    function (addons, cb) {
+      // Newline
+      print();

```

Comment:
Could you do `print('\n')` or something instead? Then you could remove the comment.